python/sdk/setup.py

```diff
@@ -36,7 +36,7 @@
     "python_dateutil>=2.5.3",
     "PyYAML>=5.4",
     "six>=1.10",
-    "urllib3>=1.23",
+    "urllib3>=1.26",
```

Comment:
Bumped up min version of `urllib3`. Context: https://github.com/caraml-dev/merlin/pull/466/files#r1356056883

Reply:
Thanks Zi Yi!